cmd/server/main.go

```diff
@@ -104,25 +106,26 @@ func fileExists(filename string) bool {
 func main() {
 	opts := &Options{}
 	flag.StringVar(&opts.bindAddr, "bind-addr", "127.0.0.1:9067", "the address to listen on")
-	flag.StringVar(&opts.dbPath, "db-path", "./database.sqlite", "the path to a sqlite database file")
 	flag.StringVar(&opts.tlsCertPath, "tls-cert", "./cert.pem", "the path to a TLS certificate")
 	flag.StringVar(&opts.tlsKeyPath, "tls-key", "./cert.key", "the path to a TLS key file")
 	flag.Uint64Var(&opts.logLevel, "log-level", uint64(logrus.InfoLevel), "log level (logrus 1-7)")
 	flag.StringVar(&opts.logPath, "log-file", "./server.log", "log file to write to")
 	flag.StringVar(&opts.zcashConfPath, "conf-file", "./zcash.conf", "conf file to pull RPC creds from")
 	flag.BoolVar(&opts.veryInsecure, "very-insecure", false, "run without the required TLS certificate, only for debugging, DO NOT use in production")
+	flag.BoolVar(&opts.noTLS, "no-tls", false, "Disable TLS, serve un-encrypted traffic.") // XXX redundant with -veryInsecure, remove
```

Comment:
It looks like this new argument had the same effect as `veryInsecure` had (my bad for the confusing name). We still want it to be really obvious that not using TLS is very insecure, so I suggest removing `veryInsecure` and renaming `no-tls`'s option to `no-tls-very-insecure`.

Reply:
Yes, these have the same effect; what happened was that the same enhancement was made to the two forks (ours and Aditya's) independently. I intentionally allowed both options because I wanted our version of lightwalletd to be backward compatible with Aditya's version, at least initially. 

But I like your suggestion, and having both is one of those things that's easy to forget about (since everything is working), and then will be much harder to fix later. I'll make your suggested change. I can coordinate with Aditya so he can make the required adjustment to his code when he switches over to using our lightwalletd.